App/UI/UploadDataAutonomous/Cells/TextFieldCun.swift

```diff
@@ -147,6 +147,11 @@ extension TextFieldCun: UITextFieldDelegate {
         if textField.text == "" {
```

Comment:
<!--
  0 failure: 
  1 warning:  swiftlint (0.39.2...
  
  
  DangerID: danger-id-Danger;
  File: App/UI/UploadDataAutonomous/Cells/TextFieldCun.swift;
  Line: 147;
-->

- :warning: swiftlint (0.39.2): Prefer checking `isEmpty` over comparing `string` to an empty string literal. (empty_string)